README.md

```diff
@@ -59,6 +59,8 @@ Finally, navigate to [app.plex.tv/desktop](https://app.plex.tv/desktop) or [your
 - `$ADVERTISE_IP`         - This variable defines the additional IPs on which the server may be be found. For example: `http://10.1.1.23:32400`. This adds to the list where the server advertises that it can be found.
 - `$DISABLE_REMOTE_SEC`   - 
 - `$PLEX_CLAIM`           - The claim token for the server to obtain a real server token. If not provided, server will not be automatically logged in. If server is already logged in, this parameter is ignored.
+- `$LOGDEBUG`             - Disables debug logging if set to 0, and defaults to 1 (enables debug logging).
```

Comment:
Pedantic but can these variables have an underscore in, e.g. `LOG_DEBUG`

Reply:
The [Plex options](https://support.plex.tv/articles/201105343-advanced-hidden-server-settings/) are `logDebug` and `LogVerbose` without spaces, so should we add an underscore?

Reply:
Typically a change in case denotes a different word. The equivalent would be an underscore in UPPER_SNAKE_CASE